packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -76,7 +81,12 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
         return
       }
 
-      ;({ handleLog: handleLogStrategy } = startLogsImpl(configuration, getCommonContext, mainLogger))
+      ;({ handleLog: handleLogStrategy, getInternalContext: hoistedGetInternalContext } = startLogsImpl(
+        configuration,
+        getCommonContext,
+        mainLogger
+      ))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Looking at what we are doing on RUM, it could be nice to keep consistent naming

cf https://github.com/DataDog/browser-sdk/blob/4590cc01553acdb191ab2d1ca68614c05afc7800/packages/rum-core/src/boot/rumPublicApi.ts#L140-L146